Libraries/ActionSheetIOS/NativeActionSheetManager.js

```diff
@@ -23,7 +24,7 @@ export interface Spec extends TurboModule {
       +destructiveButtonIndices?: ?Array<number>,
       +cancelButtonIndex?: ?number,
       +anchor?: ?number,
-      +tintColor?: ?number,
+      +tintColor?: ?ProcessedColorValue,
```

Comment:
I'm realizing that this change to the spec will require rebuilding the generated native code and making sure that the native module correctly supports the new type. 

I'm not sure how these specs need to be updated, cc @RSNara